test/unit/spec/MfaVerify_spec.js

```diff
@@ -1904,18 +1904,45 @@ function (Okta,
                 });
               });
             });
-            itp('will disable form submit', function () {
+            itp('will re-enable submit when retry timeout reached', function () {
+              spyOn(LoginUtil, 'callAfterTimeout').and.callFake(function() {
+                return setTimeout(arguments[0], 0);
+              });
               return setupOktaPush().then(function (test) {
-                return setupPolling(test, resSuccess)
-                .then(function () {
-                  expect(test.form.submitButton().attr('class')).toMatch('link-button-disabled');
-                  expect(test.form.submitButton().prop('disabled')).toBe(true);
-                  $.ajax.calls.reset();
-                  test.form.submit();
-                  return tick(test); // Final tick - SUCCESS
+                $.ajax.calls.reset();
+                test.setNextResponse([resChallengePush, resChallengePush, resSuccess]);
+                test.form = test.form[0];
+                test.form.submit();
+                expect(test.form.submitButton().attr('class')).toMatch('link-button-disabled');
+                expect(test.form.submitButton().prop('disabled')).toBe(true);
+                expect(test.form.submitButtonText()).toBe('Push sent!');
+                return tick(test);
+              })
+              .then(function (test) {
+                expect(test.form.submitButton().prop('disabled')).toBe(false);
+                expect(test.form.submitButtonText()).toBe('Re-send Push');
```

Comment:
We are checking that the button is not disabled.. Do we need to check the class as well?